config.py

```diff
@@ -0,0 +1,5 @@
+UPLOADS_FOLDER: "data/uploads"  # The path to the directory where uploaded files will be stored.
+OUTPUTS_FOLDER: "data/outputs"  # The path to the directory where output files will be saved.
+API_URL: 'http://localhost'  # The URL of the Web API.
+API_PORT: 5000  # The port on which the Web API will listen for requests.
```

Comment:
Assign constants to descriptive variables to enhance readability and simplify future changes ^__^ . Directly using them can lead to confusion and maintenance challenges.